qemu/tests/cfg/virtio_serial_file_transfer.cfg

```diff
@@ -62,6 +62,14 @@
             migration_protocol = "exec"
             migration_exec_cmd_src = "gzip -c > '%s'"
             migration_exec_cmd_dst = "gzip -c -d '%s'"
+        - abstract_tight:
+            only Linux
+            only unix_socket
+            chardev_abstract_vs1 = 'on'
+            variants:
+                 - @default:
```

Comment:
I see the purpose of this test is to cover two scenarios:
```
a). abstract=on,tight=on
b). abstract=on,tight=off
```
It is not reliable to assume the default value of tight is `on` without specify it.  Suggest specified  `chardev_tight_vs1 = 'on'` explicitly.

Reply:
> From the manual test case, I see the purpose of this test is to cover two scenarios:
> 
> ```
> a). abstract=on,tight=on
> b). abstract=on,tight=off
> ```
> 
> It is not reliable to assume the default value of tight is `on` without specify it. Suggest specified `chardev_tight_vs1 = 'on'` explicitly.

Thanks and update.